src/frontend/tests/defaults.ts

```diff
@@ -1,6 +1,6 @@
 export const classicUrl = 'http://127.0.0.1:8000';
 
-export const baseUrl = `${classicUrl}/platform`;
+export const baseUrl = `./platform`;
```

Comment:
@matmair I ran into issues using a relative URL like this - once you traverse down into lower URLs the resulting URL can become something like:

`/platform/parts/platform/stock`

Reply:
using the classic endpoint means that there is no coverage so we will have to find a middle-way